lib/tizen_tpk.dart

```diff
@@ -158,10 +158,39 @@ class TizenManifest {
 
   /// The unique application id used for launching and terminating applications.
   String get applicationId {
-    final XmlElement parent = _manifest.findElements('ui-application').first;
-    return parent.getAttribute('appid');
+    // storing _applicationId to avoid showing warning more than once
+    if (_applicationId != null) {
+      return _applicationId;
+    }
+    final Iterable<XmlElement> uiappe =
+        _manifest.findElements('ui-application');
+    final Iterable<XmlElement> srvappe =
+        _manifest.findElements('service-application');
+    if (uiappe.isEmpty && srvappe.isEmpty) {
```

Comment:
actually, if manifest does not contain any of those, the build process fails earlier so this code is not really usefull